terraform/aws/airgap/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "registry_public_dns" {
+  value = aws_instance.registry.public_dns
+}
+
+output "rke2_bastion_public_dns" {
```

Comment:
same comment here for rke2. For the outputs you can probably just remove rke2_ from each one, no need for a different name.